docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -25,7 +22,7 @@ the implementation without compat concerns in future releases.
 ### Determine target framework
```

Comment:
Fix [MD022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines).

```suggestion
### Determine target framework

```